js/bam/bamUtils.js

```diff
@@ -73,15 +74,21 @@ const BamUtils = {
             chrToIndex[name] = i
             chrNames[i] = name
 
+            if (genome) {
```

Comment:
Adding statements back in from the last working version v2.8.6 
https://github.com/igvteam/igv.js/blob/v2.8.6/js/bam/bamUtils.js#L114